www/js/search.js

```diff
@@ -918,12 +928,17 @@ module.exports = {
     while (this.$session.firstChild) {
       this.$session.removeChild(this.$session.firstChild);
       sessionList.splice(0, sessionList.length);
+      // clear object and its properties
+      Object.getOwnPropertyNames(sessionStatesList).forEach(shabadID => {
+        delete sessionStatesList[shabadID];
+      });
     }
   },
 
-  clickSession(e, ShabadID, LineID) {
+  clickSession(e, ShabadID) {
     const $panktee = e.target;
-    this.loadShabad(ShabadID, LineID);
+    const [resumePanktee] = sessionStatesList[ShabadID];
```

Comment:
not destructured correctly. should be `const { resumePanktee } = sessionStatesList[ShabadID];`